Sources/Swift/Core/Tools/ViewCapture/SentryUIRedactBuilder.swift

```diff
@@ -330,20 +506,31 @@ final class SentryUIRedactBuilder {
         // [2] https://github.com/getsentry/sentry-cocoa/blob/00d97404946a37e983eabb21cc64bd3d5d2cb474/Sources/Sentry/SentrySubClassFinder.m#L58-L84   
         let viewTypeId = type(of: view).description()
         
-        if #available(iOS 26.0, *), viewTypeId == Self.cameraSwiftUIViewClassId {
+        if #available(iOS 26.0, *), viewTypeId == Self.cameraSwiftUIViewClassId.classId {
             // CameraUI.ChromeSwiftUIView is a special case because it contains layers which can not be iterated due to this error:
             //
             // Fatal error: Use of unimplemented initializer 'init(layer:)' for class 'CameraUI.ModeLoupeLayer'
             //
             // This crash only occurs when building with Xcode 16 for iOS 26, so we add a runtime check
             return true
         }
+
+        #if os(iOS)
+        // UISwitch uses UIImageView internally, which can be in the list of redacted views.
+        // But UISwitch is in the list of ignored class identifiers by default, because it uses
+        // non-sensitive images. Therefore we want to ignore the subtree of UISwitch, unless
+        // it was removed from the list of ignored classes
+        if viewTypeId == UISwitch.self.description(), ignoreClassesIdentifiers.contains(
+            where: { $0.classId == UISwitch.self.description()
+            }) {
```

Comment:
`m`: We should be able to use `ExtendedClassIdentifier(classId: UISwitch.self.description())` to make this O(1), if I'm not mistaken.

Reply:
Replaced it to use `containsIgnoreClass` so it's reusing the same lookup/